storage/src/state/tests.rs

```diff
@@ -346,3 +354,34 @@ fn test_transaction_fees() {
     assert_eq!(new_coinbase_record.value(), expected_block_reward);
     assert_eq!(output_record.value(), amount);
 }
+
+#[test]
+fn test_get_all_ciphertexts() {
+    let rng = &mut thread_rng();
+    let terminator = AtomicBool::new(false);
+    // Initialize a new ledger.
+    let ledger = create_new_ledger::<Testnet2, RocksDB>();
+
+    // Initialize a new account.
+    let account = Account::<Testnet2>::new(&mut thread_rng());
+    let address = account.address();
+
+    // Mine the next block.
+    let (block, _) = ledger
+        .mine_next_block(address, true, &[], &terminator, rng)
```

Comment:
instead of mining, we should load a single pre-existing block for a much faster testing experience, like in the `test_get_blocks` test in `snarkos-rpc`.

Reply:
note to self: we should make sure we're not mining in _any_ test unless absolutely necessary.

Reply:
Great! I was mining a block to ensure that the function was getting all the actual ciphertexts but I think the test works well with only the genesis block.